### DISCUSSION TITLE
Add possibility to use rawProps with list of components below

### DISCUSSION THREAD

**Discussion by NatatkaValikowsky** [epam/UUI#636] [General]
Hi! We need to add HTML attributes to all components in our project to use it in auto tests.

In this components we cannot use rawProps property:
- Data Tables
- File Upload
- Paginator
- Picker List
- Rating
- Slider Rating
- Text Placeholder
- Time Picker

Do you plan to add rawProps to this components? How we can help to deal with it?

**Comment by AlekseyManetov** [comment]
Hi @NatatkaValikowsky, thanks for your research. We will add rawProps for this components in nearest releases. I created issue from this discussion - #641. If you would like to receive it earlier, you can try to implement this props functionality for all or for some of this components and contribute it to our repo.
You can find contributing guide here - https://github.com/epam/UUI/blob/main/CONTRIBUTING.md.
